Package.swift

```diff
@@ -0,0 +1,196 @@
+// swift-tools-version:5.1
+
+import PackageDescription
+
+var package = Package(name: "AwsCrt",
+                      products: [
+                        .library(name: "AwsCommonRuntimeKit",
+                                 targets: [
+                                    "AwsCCommon",
+                                    "AwsCIo",
+                                    "AwsCCompression",
+                                    "AwsCHttp",
+                                    "AwsCCal",
+                                    "AwsCAuth",
+                                    "AwsCMqtt",
+                                    "AwsCommonRuntimeKit"]),
+                        .executable(name: "Elasticurl", targets: ["Elasticurl"])
+                      ])
+
+// aws-c-common config
+var awsCCommonPlatformExcludes = ["include", "source/windows", "source/android", "AWSCRTAndroidTestRunner", "cmake", "codebuild", "docker-images", "tests", "verification"]
+//var unsafeFlagsArray: [String] = []
+
+#if arch(i386) || arch(x86_64)
+awsCCommonPlatformExcludes.append("source/arch/arm")
+//temporary cause I can't use intrensics because swiftpm doesn't like the necessary compiler flag.
+awsCCommonPlatformExcludes.append("source/arch/intel")
+//unsafeFlagsArray.append("-mavx512f")
+#elseif arch(arm64)
+awsCCommonPlatformExcludes.append("source/arch/intel")
+#else
+awsCCommonPlatformExcludes.append("source/arch/intel")
+awsCCommonPlatformExcludes.append("source/arch/arm")
+#endif
+
+#if !os(Windows)
+awsCCommonPlatformExcludes.append("source/arch/intel/msvc")
+awsCCommonPlatformExcludes.append("source/arch/arm/msvc")
+#else
+awsCCommonPlatformExcludes.append("source/arch/intel/asm")
+awsCCommonPlatformExcludes.append("source/arch/arm/asm")
+#endif
+
+var awsCIoPlatformExcludes = ["include", "tests", "cmake", "docs"]
+
+#if os(macOS)
+awsCIoPlatformExcludes.append("source/windows")
+awsCIoPlatformExcludes.append("source/linux")
+awsCIoPlatformExcludes.append("source/s2n")
+#elseif(Windows)
+awsCIoPlatformExcludes.append("source/posix")
+awsCIoPlatformExcludes.append("source/linux")
+awsCIoPlatformExcludes.append("source/s2n")
+awsCIoPlatformExcludes.append("source/darwin")
+#else
+awsCIoPlatformExcludes.append("source/windows")
+awsCIoPlatformExcludes.append("source/s2n")
```

Comment:
this should be:

awsCIoPlatformExcludes.append("source/bsd")

My fault I know

Reply:
in addtion to whats there or instead of "source/s2n"?